infra/app/database.bicep

```
@@ -71,6 +79,15 @@ var containers = [
         }
       ]
     }
+    fullTextPolicy: {
+      defaultLanguage: 'en-US',
```

Comma throwing syntax error

```suggestion
      defaultLanguage: 'en-US'
```